cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/ApplicationRequestsPlaner.java

```diff
@@ -97,20 +100,24 @@ public void visitMapChange(CfMapChange mapChange) {
     }
 
     /**
-     * Apply for all changes regarding one application. If an error occurs, the applying procedure is
+     * Creates the requests for one application. If an error occurs, the applying procedure is
      * discontinued and the app stops.
```

Comment:
The default behaviour that the procedure is discontinued on error is now changeable:
```suggestion
     * Creates the requests for one application.
```